src/styles/header/header-block.scss

```diff
@@ -0,0 +1,78 @@
+@use "./__variables-header" as *;
+@use "../cards/cards_utils/_variables" as *;
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  padding-right: 50px;
+  background-color: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  display: flex;
+  justify-self: center;
+  width: 40px;
+  height: 40px;
+  margin-left: 50px;
+  padding: 0;
+}
+
+.list {
+  display: flex;
+  justify-content: center;
+  margin: 0;
+  padding: 0;
+  list-style-type: none;
+}
+
+.link {
+  position: relative;
+  display: flex;
+  align-items: center;
+  height: 60px;
+  color: #{$main-color};
+  font-family: "Roboto", sans-serif;
```

Comment:
You don't need it here, because your FF has already in .page-content, but you need to move it in HTML tag